src/components/SelectableCard/index.tsx

```diff
@@ -55,7 +55,7 @@ const StyledElement = (element: typeof Radio) => styled(element, {
     }}
   }
 
-  svg {
+  & + svg {
```

Comment:
Can we target a styled instead ?

Reply:
Or handle this case in the Radio and Checkbox ?

Reply:
@chambo-e we cannot target styled as it is different depending if it is a radio or a checkbox, also I prefer not creating a prop into those components to display or not tick.

Also you make me notice selector is wrong it should be: 

```js
input + svg {...}
```

This way only the svg after input is removed and no others. As svg is always after input it should work in any scenario.